src/FileServiceManager/Models/FileMetadata.cs

```diff
@@ -7,9 +7,8 @@ public class FileMetadata
         public Guid Id { get; set; }
         public string Name { get; set; }
         public string Extension { get; set; }
-        public string UploadedBy { get; set; }
         public DateTime CreatedOn { get; set; }
         public string FilePath { get; set; }
+        public string FileType { get; set; }
```

Comment:
FileType is a MIME type, that is used in this constructor. For example for text files it will be text/plain. For more https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types

While extension is used to send clients and it will be .txt or .pdf or something else

Reply:
Then name it appropriately